pkg/controller/framework/informers/factory.go

```diff
@@ -27,6 +28,56 @@ import (
 	"k8s.io/kubernetes/pkg/watch"
 )
 
+type SharedInformerFactory interface {
+	Start(time.Duration)
+
+	Pod() framework.SharedIndexInformer
```

Comment:
The downstream implementation has this return an interface that gives you a `Lister` or an `Informer`.  The distinction is valuable since many locations are only interested in having the cache and we can start shaping the `Lister` API to behave more like a `ClientSet` API so the code reads consistently.